data/emulator.js

```diff
@@ -452,18 +452,16 @@ class EmulatorJS {
         this.elements.parent.appendChild(this.textElem);
     }
     localization(text, log) {
-        if (typeof text !== "undefined"){
-            text = text.toString();
-            if (text.includes("EmulatorJS v")) return text;
-            if (this.config.langJson) {
-                if (typeof log === "undefined") log = true;
-                if (!this.config.langJson[text] && log) {
-                    if(!window.EJS_missingLang) window.EJS_missingLang = [];
-                    window.EJS_missingLang.push(text);
-                    console.log("Translation not found for '"+text+"'. Language set to '"+this.config.language+"'");
-                }
-                return this.config.langJson[text] || text;
+        if (typeof text === "undefined") return;
+        text = text.toString();
+        if (text.includes("EmulatorJS v")) return text;
+        if (this.config.langJson) {
+            if (typeof log === "undefined") log = true;
+            if (!this.config.langJson[text] && log) {
+                if (!EJS_missingLang.includes(text)) EJS_missingLang.push(text);
```

Comment:
Where is EJS_missingLang defined

Reply:
On the `this` object within the constructor

Reply:
like that?

Reply:
yep, that's good